src/app/runtime/wac/components/install.component.ts

```diff
@@ -90,11 +95,33 @@ export class InstallComponent {
         this._targetHost = this.appContext.activeConnection.nodeName
     }
 
-    public verifyLocation(value: string, allowEmpty: boolean) {
-        this.invalid = value ? !urlValidationRegex.test(value) && !windowsPathValidationRegex.test(value) : !allowEmpty
+    private verifyLocation(fieldName: string, location: string, allowEmpty: boolean): string {
+        if (location) {
+            if (!urlValidationRegex.test(location) && !windowsPathValidationRegex.test(location)) {
+                return `Invalid ${fieldName}: ${location}`
+            }
+        } else {
+            if (!allowEmpty) {
+                return `${fieldName} cannot be empty`
+            }
+        }
+
+        // NOTE: This code is injected here to prevent user using shared drive because it would be considered double hop with WinRM and wouldn't work
+        // This needs to be revisited when Windows Admin Center completed their work on CredSSP
+        if (location && location.startsWith(`\\\\`)) {
+            return `Currently installation from shared drive is not supported, the issue is being tracked on https://github.com/Microsoft/IIS.WebManager/issues/239`
+        }
+
+        return null
     }
 
     public install() {
+        if (this.userInputError = this.verifyLocation(`IIS Administration API installation location`, this._adminAPILocation, false)) {
+            return
+        }
+        if (this.userInputError = this.verifyLocation(`.NET Core Runtime install location`, this._dotnetCoreLocation, true)) {
+            return

```

Comment:
put some console log here as well.

Reply:
The error shows up on screen though. Logs should be made for more implicit issues.